src/Common/tests/System/Xml/ModuleCore/cmodinfo.cs

```diff
@@ -71,7 +71,7 @@ public static string CommandLine
         {
             get
             {
-                //Deffered Parsing
+                //Defered Parsing
```

Comment:
Deferred has two r's.
(When you address PR feedback, please do it in a separate commit so I don't need to re-review 340 files again ;)